tutorial_code/example1_1_fm_loop_balancing.py

```diff
@@ -187,59 +200,101 @@ def run_experiment(param, shim, stats, embeddings, _alpha_Phi=0.0, _alpha_J=0.0)
         shim (dict): shimming data
         stats (dict): dict of sampled statistics
         embeddings (List[dict]): list of embeddings
-        _alpha_Phi (float, optional): learning rate for linear shims. Defaults to 0..
-        _alpha_J (float, optional): learning rate for coupling shims. Defaults to 0..
+        alpha_Phi (float): learning rate for linear shims. Defaults to 0.
+        alpha_J (float): learning rate for coupling shims. Defaults to 0.
+        use_cache (bool): When True an attempt is made to load (save) data from
+            (to) the directory cached_experimental_data.
+    Returns:
+       dict: experiment statistics
     """
-    prefix = f"example1_1_aPhi{_alpha_Phi}_aJ{_alpha_J}"
-
-    data_dict = {"param": param, "shim": shim, "stats": stats}
-    data_dict = load_experiment_data(prefix, data_dict)
+    if use_cache:
+        solver_name = param["sampler"].properties["chip_id"]
+        prefix = f"{solver_name}_example1_1_aPhi{alpha_Phi}_aJ{alpha_J}"
+        data_dict = {"param": param, "shim": shim, "stats": stats}
+        data_dict = load_experiment_data(prefix, data_dict)
+    else:
+        data_dict = None
 
     if data_dict is not None:
-        param = data_dict["param"]
         shim = data_dict["shim"]
         stats = data_dict["stats"]
-
     else:
-        print("Running experiment")
+        # prev_execution_time = 193.6231 sec.
```

Comment:
?
```suggestion
```
Or better reword it if it's interesting data to keep?

Reply:
Not really, Might refer to deprecated defaults as well